extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,269 @@
+@import "skins/shared/color";
```

Comment:
You might want to install pre-commit hooks from the https://github.com/Wikia/guidelines repo. Those would run `scss-lint` and tell you immediately that single quotes are preferred. But what is more important:
```
extensions/wikia/Rail/styles/RailElement.scss:55 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
extensions/wikia/Rail/styles/RailElement.scss:116 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
extensions/wikia/Rail/styles/RailElement.scss:200 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
extensions/wikia/Rail/styles/RailElement.scss:209 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
extensions/wikia/Rail/styles/RailElement.scss:225 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
extensions/wikia/Rail/styles/RailElement.scss:233 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 6
extensions/wikia/Rail/styles/RailElement.scss:236 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 7
extensions/wikia/Rail/styles/RailElement.scss:240 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 7
extensions/wikia/Rail/styles/RailElement.scss:247 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 8
extensions/wikia/Rail/styles/RailElement.scss:255 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
extensions/wikia/Rail/styles/RailElement.scss:259 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 7
```
Deep nesting of classes is lowering the page rendering performance. Maybe there's a way to lower the indentation depth? I'll look into that too and try to suggest solutions.